app/javascript/chat/components/ConnectStateProvider.jsx

```diff
@@ -12,10 +12,9 @@ const { Provider } = store;
  * @param {object} props.initialState The initial value to store as state
  * @param {Function} props.reducer Given the current state and an action, returns the new state
  */
-function ConnectStateProvider({ children, initialState, reducer }) {
-  const [state, dispatch] = useReducer(reducer, initialState);
 
+const ConnectStateProvider = ({ children, initialState, reducer }) => {
```

Comment:
This change isn't necessary. It's fine that it uses the function keyword instead of an arrow function. Arrow functions work well when you need scope, i.e. "this".